### ISSUE TITLE
Release .NET Standard

### ISSUE THREAD

**Issue by pauldendulk** [NetTopologySuite/NetTopologySuite#211]
The .NET Standard version is in pre release for a half year now. Is there a release planned? What needs to be done before it can be released?

**Comment by airbreather** [comment]
Maybe a duplicate of #206?

**Comment by pauldendulk** [comment]
@airbreather  ah thanks
